src/security/openssl/include/dds/security/openssl_support.h

```diff
@@ -38,7 +38,7 @@
 #include <WinSock2.h>
 #endif
 
-#define OPENSSL_API_COMPAT 10101
+#define OPENSSL_API_COMPAT 30000
```

Comment:
I guess this should be defined to 10101 when using OpenSSL < 3.0.0, but I checked and OpenSSL 1.1.1 doesn't care if it is set too high. Making it conditional on the OpenSSL version number seems overkill, but a comment to this effect might be useful.